app/views/templates/_form.html.erb

```diff
@@ -159,7 +159,4 @@
       not have a 'name' attribute so browsers will not send them in the form POST. %>
   <input type="hidden" id="old" value="<%= @template.template %>" />
   <input type="hidden" id="new" value="<%= @template.template %>" />
```

Comment:
I think these also need to be reordered with the submit_or_cancel so the fix is the last input